src/main/java/hometask_1/Main.java

```diff
@@ -0,0 +1,18 @@
+package hometask_1;
+
+public class Main {
+    public static void main(String[] args) throws Exception {
+
+        try {
+            for (String str : args) {
+
+                if (str.equalsIgnoreCase("error")) {
+                    System.out.println("\u001B[31m" + "Alarm!" + "\u001B[0m");
+                    throw new RuntimeException();
```

Comment:
Exception shouldn't be thrown, is't not an exceptional case, it's expected behaviour